src/test/java/com/synopsys/integration/detect/tool/impactanalysis/service/ImpactAnalysisServiceTest.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * synopsys-detect
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.detect.tool.impactanalysis.service;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.synopsys.integration.blackduck.service.BlackDuckService;
+import com.synopsys.integration.detect.testutils.TestUtil;
+import com.synopsys.integration.exception.IntegrationException;
+import com.synopsys.integration.rest.response.Response;
+
+class ImpactAnalysisServiceTest {
```

Comment:
I question that these tests provide any tangible value and are worth the 'code liability' though there is something to be said about whether or not tests are considered 'code liability'.

Reply:
I agree. I would like to add add an integration test when we have a Black Duck instance producing code locations.
It would feel weird for this tool to not have any test coverage, as there isn't any logic to test. I agree this particular test is not very useful, and in fact may be a code liability.